pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
+
+- Tech Previews can be used in Production Environments.
+- The licensing is subject to change in future versions.
```

Comment:
I don't think licensing should be mentioned in the support terms.  They have nothing to do with each other.